projects/packages/videopress/src/class-stats.php

```diff
@@ -134,7 +137,8 @@ public static function prepare_featured_stats( $dates ) {
 
 		// template for the response
 		$featured_stats = array(
-			'label' => __( 'last 7 days', 'jetpack-videopress-pkg' ),
+			// translators: %d is the number of days
+			'label' => sprintf( __( 'last %d days', 'jetpack-videopress-pkg' ), floor( $total_days / 2 ) ),
```

Comment:
What's the reason for halving the days here?

Reply:
I have no clue, it was there before and didn't want to get rid of it in case it's being used elsewhere